bridge/util.go

```diff
@@ -30,9 +30,13 @@ func combineTags(tagParts ...string) []string {
 	return tags
 }
 
-func serviceMetaData(env []string, port string) map[string]string {
+func serviceMetaData(config *dockerapi.Config, port string) map[string]string {
+	meta := config.Env
```

Comment:
I believe Go's append call will modify the array in-place if there is enough space for the new item already allocated. So it could be safer to copy the array, since it's possible that the Env array might be used again later in the caller.